services/oauth/server.go

```diff
@@ -0,0 +1,87 @@
+package oauth_services
```

Comment:
Regarding the package name, [Official Go blog](https://blog.golang.org/package-names) asks not to use underscores and  camelCase. 
> Good package names are short and clear. They are lower case, with no under_scores or mixedCaps.

How about naming it as just `oauth`?

Reply:
@junan updated in [f40ddab](https://github.com/carryall/go-google-scraper-challenge/pull/28/commits/f40ddab6e3cbe9f3269ed9eded386fd8afbe5cc9), so I think we'll only use the package name with `under_score` for testing 🤔

Reply:
@gutakk Something to check in your IC project as well and to finalize in the Go conventions 💪

Reply:
Created [issue](https://github.com/nimblehq/compass/issues/497) on compass 👍🏻